packages/forms/src/elements/checkbox/checkbox.tsx

```diff
@@ -49,7 +58,11 @@ export const Checkbox: React.FC<Partial<CheckboxIconProps>> = ({
 				)}
 				<P cfg={{ ml: 3, fontWeight: 3 }}>{label}</P>
 			</label>
-			{hint && <P className={styles.hint}>{hint}</P>}
+			{hint && (
+				<P id={id && `${id}-hint`} className={styles.hint}>
```

Comment:
Can/should the id be helper.hintId?

Reply:
Good spot. Left from a previous iteration